hwy/ops/x86_256-inl.h

```diff
@@ -2207,6 +2207,18 @@ HWY_API Vec256<float> ApproximateReciprocal(Vec256<float> v) {
   return Vec256<float>{_mm256_rcp_ps(v.raw)};
 }
 
+#if HWY_TARGET <= HWY_AVX3
```

Comment:
Are we sure this behaves as intended? Isn't it enough to toggle the HWY_NATIVE flag only in x86_128 because that is included from all other x86 headers? I think we can skip this toggle here.